app/services/jobs/return-logs/check-return-cycle.service.js

```diff
@@ -11,6 +11,7 @@ const {
   determineCycleEndDate,
   determineCycleStartDate
 } = require('../../../lib/return-cycle-dates.lib.js')
+const FeatureFlagsConfig = require('../../../../config/feature-flags.config.js')
```

Comment:
This needs to go below the other `requires()`. See [Top of .js files](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files)

```suggestion
const featureFlagsConfig = require('../../../../config/feature-flags.config.js')
```